docs/README.adoc

```diff
@@ -4,7 +4,7 @@
     generator or the input template (./*.in)
 ////
 
-= Application Templates for Red Hat Single Sign-On 7.3 continuous delivery (CD) Tech Preview OpenShift container image
+= Application Templates for Red Hat Single Sign-On Continuous Delivery (CD) Tech Preview OpenShift container image
```

Comment:
You don't need to change this file directy. Once you change [README.adoc.in](https://github.com/jboss-container-images/redhat-sso-7-openshift-image/blob/sso-cd-dev/README.adoc.in) file and rerun `$ ./gen_template_docs.py ` the change will get propagated into all `docs/*.adoc` files too.

Reply:
I changed both. If you look into `README.adoc.in`, you will notice that it also contains the same changes. I'm trying to use mass replace whenever I can, so in most of the places changes should be very similar.